web/micro-ui-internals/packages/modules/fsm/src/pages/employee/GarimaDetails/index.js

```diff
@@ -0,0 +1,200 @@
+    const history = useHistory();
+
+
+    const { data: vehicleList, isLoading: isVehicleData, isSuccess: isVehicleDataLoaded } = Digit.Hooks.fsm.useMDMS(
+        stateCode,
+        "Vehicle",
+        "VehicleType",
+        { staleTime: Infinity }
+    );
+
+    const { isLoading: isDataLoading, isSuccess, data: applicationData } = Digit.Hooks.fsm.useSearch(
+        tenantId,
+        { applicationNos: applicationNumber },
+        { staleTime: Infinity }
+    );
+
+    const {
+        isLoading: isDriverLoading,
+        isError: driverUpdateError,
+        data: updateDriverResponse,
+        error: updateDriverError,
+        mutate: mutateDriver,
+    } = Digit.Hooks.fsm.useApplicationUpdate(tenantId);
+
+    
+
+    // const {
+    //     isLoading: isDriverLoading,
+    //     isError: isDriverError,
+    //     data: driverData,
+    //     error: driverError,
+    //     mutate,
+    // } = Digit.Hooks.fsm.useGarimaSearchActions(tenantId);
+
+    const onSubmit = (data) => {
+        let sanitationDetails = [data.garimaDriverDetails,...data.garimaHelperDetails.helperList];
+        
+        const createPayload = {
+            "fsm":{
+                ...applicationData,
+                sanitationWorker:sanitationDetails,
+                vehicleId: data.vehicleNumber.id,
+            },
+            "workflow": {
+                "action": "DSO_ACCEPT"
+            },
+            
+        };
+        mutateDriver(createPayload, {
+            onError: (error, variables) => {
+              setShowToast({ key: "error", action: error?.response?.data?.Errors[0].message});
+              setTimeout(closeToast, 5000);
+            },
+            onSuccess: (data, variables) => {
+                setShowToast({ key: "success", action: "CS_COMMON_TRACK_APPLICATION_TEXT"});
+                history.push("/digit-ui/employee/fsm/application-details/" + applicationNumber)
+            },
+        });
+    };
+
+    const closeToast = () => {
+		setShowToast(null);
+	};
+
+    useEffect(()=>{
+        setGarimaApplicatinData(applicationData)
+    },[isSuccess]) 
+    
+
+    const onFormValueChange = (setValue, formData) => {
+    };
+    
+    // const searchGarimaWorkerDetails = (data, configKey, formData, onSelect) => {
+    //     mutate(data, {
+    //       onError: (error, variables) => {
+    //         setShowToast({ key: "error", action: error });
+    //         setTimeout(closeToast, 5000);
+    //       },
+    //       onSuccess: (data, variables) => {
+    //         setGarimaDriverData (data)
+    //         let garimaDriverDetails = formData.garimaDriverDetails || {};
+    //         garimaDriverDetails.garimaDriverMobileNumber = data[0].mobile_number;
+    //         garimaDriverDetails.garimaDriverName =  `P${data[0].garima_id}`;
+    //         onSelect(configKey, { ...formData[configKey], garimaDriverDetails: garimaDriverDetails });
+    //       },
+    //     });
+    // };
+
+    const configs = [
```

Comment:
Create new config file for this and import @chinmoy-eGov

Reply:
We are sending hook data through config, so we can not create a new config for this case

Reply:
ok